### ISSUE TITLE
Paper results use 100 epochs ~= 126k iterations. These results use 160k iterations?

### ISSUE THREAD

**Issue by cjrd** [implus/mae_segmentation#2]
The paper reports results for 100 epochs of training with a batch size of 16. For the 20,210 ade20k training images this is 20,210x100/16 ~= 126k iterations. I noticed your results use 160k iterations -- any idea if this reproduces the results with 100 epochs?

**Comment by implus** [comment]
Hi, you can refer to the log in this repo which shows 47.7 mIoU results very close to 48.1 at iteration 128k with relatively large learning rate at that time. I suspect 128k is enough to obtain ~48.1 result if you just modify the total iterations from 160k to 128k (or 126k). You can have a try.
